### DISCUSSION TITLE
Option to put Images/Embeds/Tables in full width still using boxed width page

### DISCUSSION THREAD

**Discussion by matbrgz** [outline/outline#3104] [Ideas]
Within a limited width document I would like to allow some content types like images, embeds and tables to take up a little more of the screen. Certain types of content require a little more space to fit together better. It doesn't necessarily need to be 100%, but 60% would be perfect (maybe one day it won't allow customizing the window size).

![image](https://user-images.githubusercontent.com/23284276/153730173-e207b33f-81a3-4067-b577-a5e018417c73.png)
Image from Medium

**Comment by therajumandapati** [comment]
Confluence does an excellent job with this, and it's very useful, especially when tables have 4 or more columns.

**Reply by therajumandapati** [reply]
@tommoor If you could point me in the right direction, I may be able to assist with adding this feature.

**Reply by tommoor** [reply]
The problem is that all document state must still be serializable to Markdown, in the future this won't be the case but it still is today – so you need a way to represent this "full width" option that can be serialized and parsed from markdown. I'm not sure this is possible and it's likely blocked by #3000

**Reply by therajumandapati** [reply]
I see. How much work do you think is involved in #3000?

**Reply by spacec0de** [reply]
I don't know if this is of any value, but [this guy ](https://minimal.guide/Features/Block+width)did it in Obsidian with [this plugin](https://github.com/kepano/obsidian-minimal#plugin-support) and somehow based it on [this plugin](https://github.com/mgmeyers/obsidian-contextual-typography).
